client/src/Movies/Movie.js

```diff
@@ -11,7 +11,10 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
```

Comment:
You can take out this line since you are now using the match obj for the id param